sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1238,6 +1238,15 @@ object SQLConf {
       .booleanConf
       .createWithDefault(true)
 
+  val COMBINE_LIMIT_AFTER_SORT_THRESHOLD =
+    buildConf("spark.sql.execution.combineLimitAfterSortThreshold")
```

Comment:
after some more thoughts, this config name is not very obvious. How about `topNSortFallbackThreshold`? any better ideas?

Reply:
topK

Reply:
or even simpler, `topKThreshold`

Reply:
`topK` seems to be selected from many samples. 
How about `limitKSortFallbackThreshold` or `limitKThreshold`?

Reply:
I go by `topKSortFallbackThreshold` and updated the change.